todoman/model.py

```diff
@@ -587,6 +589,8 @@ def todos(self, all=False, lists=[], urgent=False, location='',
             params.extend(lists)
         if urgent:
             extra_where.append('AND priority = 9')
+        elif priority:
+            extra_where.append('AND priority = ' + priority)
```

Comment:
Yes, I did but I was just asking what would be a better approach.
I'll change the documentation accordingly then.
What do you think would be a better way?